components/PageBanner/PageBanner.styled.tsx

```diff
@@ -0,0 +1,38 @@
+import styled from '@emotion/styled'
+import { breakpoint } from 'components/utils/styles/breakpoints'
+import { calcRem } from 'components/utils/styles/calcRem'
+
+interface StyledPageBanner {
+  banner_image?: string
```

Comment:
minor: most of our props are `camelCase`

Reply:
Good call, I thought I had cleaned all this up! The perils of switching back and forth between projects using camelCase and snake_case...